doc/script_commands.txt

```diff
@@ -7773,6 +7773,17 @@ OnAtcommand:
 
 ---------------------------------------
 
+*add_group_command("<command>", <group id>, <use on self>, <use on other>)
+
+Allows to explicitly change the command permissions for a specific group.
```

Comment:
As I mentioned in #1726 can you point out explicitly that the group inheritance is disregarded?